Source: Upstream git.
Upstream: Yes.
Reason: linux-headers-2.6.32 broke strace compilation:

net.c: In function 'printsock':
net.c:976: error: field 'nl' has incomplete type

From f0df31e71a58c6e79ba77c1a9d84b2f38d44bec7 Mon Sep 17 00:00:00 2001
From: Dmitry V. Levin <ldv@altlinux.org>
Date: Tue, 3 Nov 2009 16:49:49 +0000
Subject: [PATCH] Fix check for linux/netlink.h on Linux 2.6.32-rc5+

* configure.ac (AC_CHECK_HEADERS): In check for linux/netlink.h, include
sys/socket.h instead of linux/socket.h beforehand.
---
 configure.ac |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/configure.ac b/configure.ac
index c8450cd..96c1c24 100644
--- a/configure.ac
+++ b/configure.ac
@@ -258,7 +258,7 @@ AC_CHECK_HEADERS([ \
 ], [], [])
 AC_CHECK_HEADERS([linux/icmp.h linux/in6.h linux/netlink.h linux/if_packet.h],
                  [], [], [#include <stddef.h>
-#include <linux/socket.h>])
+#include <sys/socket.h>])
 AC_CHECK_HEADERS([asm/sigcontext.h], [], [], [#include <signal.h>])
 AC_CHECK_TYPES([struct sigcontext_struct,
 		struct sigcontext],,, [#include <signal.h>])
-- 
1.6.5.5

